thorlcr/activities/group/thgroupslave.cpp

```diff
@@ -61,6 +61,8 @@ class GroupSlaveActivity : public CSlaveActivity
         numGroups = 0;
         numGroupMax = 0;
         startLastGroup = 0;
+        if (container.queryLocalOrGrouped())
```

Comment:
similar question here - why does this require initData?  because it contains the rows from the next node?